front/app/components/SignUpIn/SignUpInModal.tsx

```diff
@@ -60,6 +61,10 @@ const SignUpInModal = memo<Props>(
         ? 820
         : 580;
 
+    const isFranceconnectLoginEnabled = useFeatureFlag({
+      name: 'franceconnect_login',
+    });
+
```

Comment:
FYI, regardless of what the solution will be like, we can't do this. France_connect logic should not be in the core app.

Reply:
(If you find it in other places in the core, that's wrong too.)